File: packages/jexlang-java/src/main/java/com/jexlang/java/eval/errors/DivisionByZeroError.java
package com.jexlang.java.eval.errors;

public class DivisionByZeroError extends JexLangRuntimeError {
    public DivisionByZeroError() {
        super("Division by zero");
    }

    @Override
    public String getName() {
        return "DivisionByZeroError";
    }
}

File: packages/jexlang-swift/jexlang-swift/eval/errors/DivisionByZeroError.swift
//
//  DivisionByZeroError.swift
//  jexlang-swift
//
//  Created by Vinay Kumar on 03/10/25.
//

import Foundation

public class DivisionByZeroError: JexLangRuntimeError {
    public init() {
        super.init(message: "Division by zero");
    }

    public override func getName() -> String {
        return "DivisionByZeroError";
    }
}
